typescript/formsApps.d.ts

```diff
@@ -503,7 +505,7 @@ export type FormsAppConfiguration<
   isGoogleLoginSupported: boolean
   isClientLoggingEnabled: boolean
   recaptchaPublicKey: string
-  googleMapsIntegrationKeyId?: string
+  googleMapsApiKey?: string
```

Comment:
I think we would be better off not making this optional and setting it in our API to the default if they dont have a an integration setup

Reply:
I'll do it now to master.